app/src/main/java/org/sopt/dosopttemplate/data/model/response/ResponseUserDto.kt

```diff
@@ -0,0 +1,50 @@
+package org.sopt.dosopttemplate.data.model.response
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.sopt.dosopttemplate.domain.entity.UserEntity
+
+@Serializable
+data class ResponseUserDto(
+    @SerialName("page")
+    val page: Int = 0,
+    @SerialName("per_page")
+    val per_page: Int = 0,
```

Comment:
변수명을 스네일 케이스보다는 카멜 케이스를 사용해야 합니다 ~~
```suggestion
    @SerialName("per_page")
    val perPage: Int = 0,
```